containers/home.tsx

```diff
@@ -1,66 +1,89 @@
-import React from 'react';
-import Head from 'next/head'
-import styles from '../styles/Home.module.css'
+import React, { useState, useEffect } from 'react';
+import Head from 'next/head';
+
+// Components
+import VideoCardList from '../components/videoCardList/videoCardList';
+import ArrowPrevSVG from '../components/svg/arrowPrevSVG';
+import ArrowNextSVG from '../components/svg/arrowNextSVG';
+
+// Style
+import styles from '../styles/Home.module.css';
+
+interface iVideo {
+  cards?: [{ title: string; alt: any; src: any }];
+}
+
+const Home: React.FC<iVideo> = () => {
+  const [currentVideoIndex, setCurrentVideoIndex] = useState(0);
+  const [videosToShow, setVideosToShow] = useState([]);
+
+  const cards = [
```

Comment:
You can receive this array as an argument, and then in the storybook you can copy these elements